projects/container_bench/testing/test_container_bench.py

```
@@ -58,22 +62,158 @@ def test():
+        json_benchmark_file = matbenchmark.prepare_benchmark_file(
+            path_tpl="test",
+            script_tpl=f"{sys.argv[0]} matbench_run",
+            stop_on_error=config.project.get_config("test.matbenchmarking.stop_on_error"),
+            common_settings=dict(),
+            test_files={},
+            expe_to_run=expe_to_run,
+        )
+
+        logging.info(f"Benchmark configuration to run: \n{yaml.dump(json_benchmark_file, sort_keys=False)}")
+
+        benchmark_file, yaml_content = matbenchmark.save_benchmark_file(json_benchmark_file)
```

Ruff (0.13.2)

Unpacked variable `yaml_content` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)